scripts/update-internal-dependencies.js

```diff
@@ -0,0 +1,85 @@
+const fs = require("fs").promises;
+
+const semver = require("semver");
+
+// get cli arguments. Any cli argument is assumbed to be the name of a package
+// dependency to update in other packages. If no extra cli arguments are
+// provided, then all packages will be updated.
+//
+// For example:
+//   `yarn update-internal-dependencies`
+//      updates everything
+//   `yarn update-internal-dependencies core-utils base-map`
+//      updates packages that depend on either @opentripplanner/core-utils or
+//      @opentripplanner/base-map
+const filteredPackages = process.argv.slice(2);
+if (filteredPackages.length === 0) {
+  console.log("Updating all internal @opentripplanner dependencies!");
+}
+
+/**
+ * Updates all internal @opentripplanner/* dependencies in each package's
```

Comment:
Who or what should normally trigger this new script? The user or the CI build?

Reply:
This should be manually run by a user.

Reply:
I think this should be captured in comments and also documented in the README in a new section about Releases.

Reply:
I modified the project's README in 5163a219c2ab5072d9338041435293fda8877c42